src/common/hooks/useImageSize.ts

```diff
@@ -145,7 +137,7 @@ export const useImageSize = <
     if (load && getPreviousId() !== id) {
       setPreviousId(id)
       // Request the desired size
-      dispatch(action(id, size))
+      dispatchProp ? dispatchProp(action(id, size)) : dispatch(action(id, size))
```

Comment:
Did this fix it on mobile? I feel like it might still break because we still have `useDispatch`

Reply:
Did some testing, the error still appears to be popping up. Will do some more testing tomorrow, but this may not have worked 😞

Reply:
Removing the `useDispatch` will hopefully fix it, we will need to explicitly pass in dispatch on web